benchmarks/logged_aiperf.py

```
@@ -0,0 +1,241 @@
+    args = p.parse_args()
+
+    keep_artifacts = args.artifact_dir is not None
+    if args.artifact_dir is None:
+        args.artifact_dir = "artifacts"
+
+    # Parse raw strings
+    ai_kv, ai_bare = parse_kv_string(args.aiperf_args)
+    wb_kv, wb_bare = parse_kv_string(args.wandb_args)
+
+    # Build aiperf command
+    cmd = ["aiperf", "profile"] + kv_to_aiperf_cmd(ai_kv, ai_bare)
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)